zkevm-circuits/src/rlp_circuit_fsm/util.rs

```diff
@@ -0,0 +1,83 @@
+use eth_types::Field;
```

Comment:
Because the `IsZeroChip` provided by the gadgets module can only give us `is_zero_expression` at current row. In the case of RLP, it will be desireable to access the `is_zero_expression` at any rotation.